kaos/algorithm/__init__.py

```diff
@@ -3,7 +3,16 @@
 from collections import namedtuple
 
 Vector3D = namedtuple( "Vector3D", "x,y,z" )
-
+TimeInterval = namedtuple( "TimeInterval", "start,end")
 # WGS84 ellipsoid constants:
 ELLIPSOID_A = 6378137
 ELLIPSOID_E = 8.1819190842622e-2
+
+# Viewing cone calculation constants:
+ANG_VEL_EARTH =  7.2921159e-5
+SECONDS_PER_DAY = 86164 #23:56:04
+THETA_NAUGHT = 0 #visibility threshold
+
+class ViewConeFailure(Exception):
```

Comment:
Strongly agree -- this improves usability for any 3rd parties in the future who might want to `import kaos` and have access to all the exceptions easily.

I didn't bring it up here because I'd put it in the refactoring plans already: https://trello.com/c/B80Ykj7T

Reply:
Oh, I actually have not checked Trello in a while. I see it now.

Reply:
Updated the name, Added a TODO